features/nanostack/FEATURE_NANOSTACK/mbed-mesh-api/source/thread_tasklet.c

```diff
@@ -209,37 +209,37 @@ void thread_tasklet_parse_network_event(arm_event_s *event)
     }
 }
 
-/*
- * \brief Configure mesh network
- *
- */
-void thread_tasklet_configure_and_connect_to_network(void)
-{
-    int8_t status;
-
-    if (MBED_MESH_API_THREAD_DEVICE_TYPE == MESH_DEVICE_TYPE_THREAD_SLEEPY_END_DEVICE) {
-        thread_tasklet_data_ptr->operating_mode = NET_6LOWPAN_SLEEPY_HOST;
-    } else {
-        thread_tasklet_data_ptr->operating_mode = NET_6LOWPAN_ROUTER;
-    }
-
-    arm_nwk_interface_configure_6lowpan_bootstrap_set(
-        thread_tasklet_data_ptr->nwk_if_id,
-        thread_tasklet_data_ptr->operating_mode,
-        NET_6LOWPAN_THREAD);
+void read_link_configuration() {
+    
+    memcpy(thread_tasklet_data_ptr->link_config.name, "Arm Powered Core", 16);
```

Comment:
What does "Arm powered core mean"?

Should probably use sizeof() instead of an int for size